x/gamm/keeper/msg_server.go

```diff
@@ -215,7 +215,7 @@ func (server msgServer) JoinSwapExternAmountIn(goCtx context.Context, msg *types
 		return nil, err
 	}
 
-	tokensIn := sdk.Coins{msg.TokenIn}
+	tokensIn := sdk.NewCoins(msg.TokensIn...)
```

Comment:
Is it possible to do the following?
```
tokensIn := msg.TokensIn
```

Reply:
Or we can just directly use msg.TokensIn as an argument?